src/engine-scripts/puppet/scrollToc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = async ( page ) => {
+	await page.evaluate( () => {
+		const tocElement = document.getElementById( 'vector-toc' );
+		const tocToggleBtns = tocElement.querySelectorAll( '.vector-toc-toggle' );
+		for ( const btn of tocToggleBtns ) {
+			btn.click();
+		}
+		tocElement.scrollBy( 0, tocElement.scrollHeight );
+		// Scroll up 30px from the bottom of the TOC to allow the scrollable indicator to show
+		tocElement.scrollBy( 0, -30 );
```

Comment:
We might need to wait a number of frames after the scroll as I noticed one that test failed for me because it seems that the new scroll hadn't been painted yet [1]

<img width="2534" alt="Screen Shot 2023-01-13 at 3 36 32 PM" src="https://user-images.githubusercontent.com/3496279/212432943-6ef87843-b698-4be5-8cb6-0e5dbf3e2b54.png">